acstech/doppler-events
# Peterk/couchbase

Made sure that the API will only accept data from clients who exist (have a couchbase document) and that eventID's are stored in the couchbase bucket of the given client.
Fixes #41 
Adds functionality such as making sure a client exists and making sure that the eventID exists via the couchbase bucket.

Testing:
1) start docker: `docker-compose up -d`
2) wait 10 seconds then go to http://localhost:8091/, setup couchbase giving it 256 MB for memory in advanced settings, and create an admin user.
3) go to the bucket tab on the upper left nav bar and create a bucket (upper right corner) called doppler
4) go to the security tab on the bottom left nav bar and create a user called validator with the password rotadilav and read and write permissions on doppler
5) go to the buckets tab again and add a document called doppler:client:test2
6) run `go run cmd/grpcTEST/serviceStart.go` if there is an error, try getting gocb 
7) then run `go run cmd/testsend/setsSend.go`
8) next check influx for the data sent (`docker-compose exec influx_db influx`, USE doppler, SELECT * FROM dopplerDataHistory)
